### ISSUE TITLE
Add keyboard shortcuts to improve navigation and accessibility

### ISSUE THREAD

**Issue by alopezlago** [alopezlago/MODAQ#27]
Add keyboard shortcuts to improve navigation and accessibility. Some improvements would be:
- In the New Game dialog, there should be a shortcut to move between teams.
- Clicking on down/up in a player entry field in the New Game dialog should move focus to the next/previous textbox, respectively
    - If there are no players below and the user hits the down key, we should add a user
- For accessibility, there should be a way to move to a word in the tossup, and a key they can press (outside of the menu key that comes with some keyboards) to bring up the buzz menu.
